repository/SmalltalkCI-Core.package/SCIMetacelloLoadSpec.class/instance/ignorePackageNotificationClass.st

```diff
@@ -0,0 +1,6 @@
+executing
+ignorePackageNotificationClass
```

Comment:
You need to dispatch this to `SmalltalkCI platformClass`. `#at:` raises `KeyNotFound` in Pharo 13, so you need to use `#at:ifAbsent:` there, but `#at:` in other cases.

Reply:
Updated. It was a bit more complex than that (I had to re-launch a P3 32-bit image to understand) but now the build is green.